ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -0,0 +1,289 @@
+{
+    WT_UNUSED(storage_source)
+    return 0;
+}
+
+static int
+AzureFileSystemTerminate(WT_FILE_SYSTEM *file_system, WT_SESSION *session)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    return 0;
+}
+
+static int
+AzureFlush(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session, WT_FILE_SYSTEM *file_system,
+  const char *source, const char *object, const char *config)
+{
+    WT_UNUSED(storage_source);
+    WT_UNUSED(session);
+    WT_UNUSED(file_system);
+    WT_UNUSED(source);
+    WT_UNUSED(object);
+    WT_UNUSED(config);
+    return 0;
+}
+
+static int
+AzureFlushFinish(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session,
+  WT_FILE_SYSTEM *file_system, const char *source, const char *object, const char *config)
+{
+    WT_UNUSED(storage_source);
+    WT_UNUSED(session);
+    WT_UNUSED(file_system);
+    WT_UNUSED(source);
+    WT_UNUSED(object);
+    WT_UNUSED(config);
+    return 0;
+}
+
+static int
+AzureObjectList(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *directory,
+  const char *prefix, char ***dirlistp, uint32_t *countp)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    WT_UNUSED(directory);
+    WT_UNUSED(prefix);
+    WT_UNUSED(dirlistp);
+    WT_UNUSED(countp);
+    return 0;
+}
+
+static int
+AzureObjectListSingle(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *directory,
+  const char *prefix, char ***dirlistp, uint32_t *countp)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    WT_UNUSED(directory);
+    WT_UNUSED(prefix);
+    WT_UNUSED(dirlistp);
+    WT_UNUSED(countp);
+    return 0;
+}
+
+static int
+AzureObjectListFree(
+  WT_FILE_SYSTEM *file_system, WT_SESSION *session, char **dirlist, uint32_t count)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    WT_UNUSED(dirlist);
+    WT_UNUSED(count);
+    return 0;
+}
+
+static int
+AzureFileSystemTerminate(WT_FILE_SYSTEM *file_system, WT_SESSION *session)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    return 0;
+}
+
+static int
+AzureFileExists(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, bool *existp)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    WT_UNUSED(name);
+    WT_UNUSED(existp);
+    return 0;
+}
+
+static int
+AzureRemove(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, uint32_t flags)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    WT_UNUSED(name);
+    WT_UNUSED(flags);
+    return 0;
+}
+
+static int
+AzureRename(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *from, const char *to,
+  uint32_t flags)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    WT_UNUSED(from);
+    WT_UNUSED(to);
+    WT_UNUSED(flags);
+    return 0;
+}
+
+static int
+AzureObjectSize(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, wt_off_t *sizep)
+{
+    WT_UNUSED(file_system);
+    WT_UNUSED(session);
+    WT_UNUSED(name);
+    WT_UNUSED(sizep);
+    return 0;
```

Comment:
we usually encase a bracket for return values.

Reply:
done